.github/workflows/Build.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Set workflow & branch variables
         id: set-variables
         run: |
-          export TIMESTAMP="$(date -u +%F-%H-%M)"
+          export TIMESTAMP="$(date -u +%F-%H)Z"
```

Comment:
Shouldn't the `Z` be inside the date format?

Reply:
Switched it to 
```
date -u +%F-%H%Z
2024-12-13-19UTC
```